students/jinsungpark/users/views.py

```diff
@@ -43,14 +45,20 @@ def post(self, request):
 class LogInView(View):
     def post(self, request):
         try:
-            data      = json.loads(request.body)
-            email     = data["email"]
-            password  = data["password"]
+            data  = json.loads(request.body)         
+            user  = User.objects.get(email=data["email"])
+            if bcrypt.checkpw(data["password"].encode('utf-8'), user.password.encode('utf-8')):
+                access_token = jwt.encode({'id' : user.id}, SECRET_KEY, algorithm=ALGORITHM)
```

Comment:
if문으로 비밀번호가 틀렸을 경우를 걸러주고, 그 이후에 잘 통과된 경우 success를 리턴해주는게 더 좋아보입니다 진성님!
```suggestion
            if not bcrypt.checkpw(data["password"].encode('utf-8'), user.password.encode('utf-8')):
                return JsonResponse({"message" : "invalid_password"}, stauts=401)
```

Reply:
수정완료했습니다